package.json

```diff
@@ -34,8 +34,8 @@
     "request": "^2.88.0",
     "session.socket.io": "^0.1.6",
     "settings-sharelatex": "^1.1.0",
-    "socket.io": "0.9.19",
-    "socket.io-client": "^0.9.16"
+    "socket.io": "2.3.0",
+    "socket.io-client": "2.3.0"
```

Comment:
Could `-client` be a devDependency?

Reply:
I am not sure. We want to serve the client side js in this particular version in production. Previously we diverged from `0.9.19` (backend)  and `0.9.16` (frontend).

socket.io depends on it in the matching version, but this might change at some-point?
https://github.com/socketio/socket.io/blob/47161a65d40c2587535de750ac4c7d448e5842ba/package.json#L31
The serving logic checks for our socket.io client blob first, before leveraging the `require` logic.
https://github.com/socketio/socket.io/blob/47161a65d40c2587535de750ac4c7d448e5842ba/lib/index.js#L105-L124

Reply:
OK thanks - I thought it would have got it via webpack from the frontend.